ote/src/clj/ote/services/transit_changes.clj

```diff
@@ -111,26 +113,35 @@
          (http/transit-response (detection/hash-recalculations db))))
 
   ;; Calculate date-hashes. day/month/contract (all or only latest on every month or only for contract traffic) true/false (only to future or all days)
-  (GET "/transit-changes/hash-calculation/:scope/:future" [scope future :as {user :user}]
+  (GET "/transit-changes/hash-calculation/:scope/:future" [scope is-future :as {user :user}]
     (when (authorization/admin? user)
+      ;; Start slow process in other thread
+      (future
+        (cond
+          (= scope "month")
+          (detection/calculate-monthly-date-hashes-for-packages
+            db (:user user) (= "true" is-future))
+          (= scope "day")
+          (detection/calculate-date-hashes-for-all-packages
+            db (:user user) (= "true" is-future))
+          (= scope "contract")
+          (detection/calculate-date-hashes-for-contract-traffic
+            db (:user user) (= "true" is-future))))
+      ;; But give thumbs up
       (http/transit-response
         {:status 200
-         :body (cond
-                     (= scope "month")
-                     (detection/calculate-monthly-date-hashes-for-packages
-                       db (:user user) (= "true" future))
-                     (= scope "day")
-                     (detection/calculate-date-hashes-for-all-packages
-                       db (:user user) (= "true" future))
-                     (= scope "contract")
-                     (detection/calculate-date-hashes-for-contract-traffic
-                       db (:user user) (= "true" future)))})))
+         :body "OK"})))
```

Comment:
If we want to be exact, result in this case should be `202 Accepted` indicating processing is started but not completed.